data/config_options.yml

```
@@ -103,6 +107,12 @@ options:
         ```
 
         -> **Note**: Changing the [name](#option-name) or environment of an existing app will create a new app on AppSignal.com.
+    nodejs:
+      config_key: appEnv
```

```suggestion
      config_key: environment
```

Should be `environment` according to this map: https://github.com/appsignal/appsignal-nodejs/blob/a75027dacc3ba1920e579218350f64c09a606d1b/packages/nodejs/src/config/configmap.ts#L5